src/transaction_builder.cpp

```diff
@@ -524,6 +524,10 @@ void TransactionBuilder::CreateJSDescriptions()
             // Update tree state with previous joinsplit
             SproutMerkleTree tree;
             {
+                // assert that coinsView is not null
+                assert(coinsView);
+                // We do not check cs_coinView becaue we do not set this in testing
+                // assert(cs_coinsView);
```

Comment:
It seems we want to assert here, but because the tests break, we are not going to.  Which puts us back in the pre-PR position of not having an assertion, which contributed to the default. I suggest filing a ticket to resolve this post-release unless the tests can be updated to pass with the assertion in place.

Reply:
I agree that we do want to be asserting this. I will file a ticket to fix this post release.